man/man8/zstream.8

```diff
@@ -82,6 +85,20 @@ alias is provided for compatibility and is equivalent to running
 Dumps zfs resume token information
 .It Xo
 .Nm
+.Cm decompress
+.Oo Ar inode Ns Sy \&, Ns Ar offset Oc Ns ...
+.Xc
+Decompress selected records in a ZFS send stream provided on standard input.
+This allows recovering a ZFS dataset that was corrupted by OpenZFS bug #12762.
```

Comment:
Being able to decompress records in the stream seems generically useful to me beyond bug #12762.  Personally, I'd suggest dropping this sentence entirely and instead add an example to how this can be used to repair bug #12762.  Take a look at the `zdb(8)` man page for how to set up an EXAMPLES section for the man page.

Reply:
What other use do you envision?

Reply:
Admittedly nothing specific, but it seems a shame not to make this functionality a bit more general since it's already so close.